src/applications/vaos/referral-appointments/utils/appointment.js

```diff
@@ -1,24 +1,37 @@
 const appointmentData = {
-  id: 'yyQYn7be',
+  id: 'QC9S60he',
   type: 'epsAppointment',
   attributes: {
-    id: 'yyQYn7be',
+    id: 'QC9S60he',
     status: 'booked',
     start: '2024-11-18T13:30:00Z',
-    isLatest: false,
-    lastRetrieved: '2025-01-29T16:30:25Z',
+    past: false,
+    isLatest: true,
+    lastRetrieved: '2025-10-14T15:15:07Z',
     modality: 'communityCareEps',
-    past: true,
     provider: {
-      id: 'DBKQ-123',
+      id: 'KMWjsCY3',
+      name: 'Dr. Smith @ Acme Cardiology - Anywhere, USA',
```

Comment:
We should check to see if the swagger is accurate here. If it will always append the location name with an @ symbol we would probably want to trim that. Since we are showing it below. We could also go with using the 
```
"individualProviders": [
        {
          "name": "Dr. Smith",
          ...
```